Sprint-3/slideshow/slideshow.js

```diff
@@ -1,8 +1,50 @@
 const images = [
-    "./assets/cute-cat-a.png",
-    "./assets/cute-cat-b.jpg",
-    "./assets/cute-cat-c.jpg",
+  "./assets/dog1.webp",
+  "./assets/dog2.webp",
+  "./assets/dog3.avif",
+  "./assets/dog5.webp",
 ];
+let i = 0;
+let autoCount;
```

Comment:
What value does `setInterval()` return? Can you come up with a more appropriate identifier to store its return value?